src/node/rpc/base_endpoint_registry.h

```diff
@@ -66,6 +78,8 @@ namespace ccf
       }
     }
 
+    /** Get the ID of latest transaction known to be committed.
+     */
     std::string get_last_committed_txid_v1(
```

Comment:
Why go with returning an error string over returning the data type and throwing on error?